integration-tests/README.md

```diff
@@ -0,0 +1,16 @@
+## Steps to Run Locally
+1. Install docker (if not already installed)
+2. Open a command prompt
+3. Clone this repository
+4. `cd <REPO_ROOT>/integration-tests`
+5. `docker compose --project-directory docker-sws-api up --force-recreate`
+
+## Notes
+* We use docker for dev testing because ES will no longer run on higher Java versions, so this is the easiest way to get it up and running.
+* .NET running locally on a Mac cannot talk to ES because of how NEST always uses the host name to connect to ES and ES exposes the Virtual Machine's hostname/IP that runs Linux on the Mac.
+* The `docker up` command above omits the "detached" (`-d`) option to make the containers easier to stop. It may be run either way.
+* In order to get the API rebuilt after changes to the code, you must run
+  * `cd <REPO_ROOT>/integration-tests`
+  * `docker compose down --rmi local --project-directory docker-sws-api`
+  * `docker compose up -d --project-directory docker-sws-api`
+  * `bin/load-integration-data.sh`
```

Comment:
This is missing a `docker compose build` step.  Sorry I missed that the previous go-round.

Reply:
Hmm. How would that be needed, given that the `up` command will automatically build the missing pieces, and that everything local will have been removed by `--rmi local`?

I do need to modify the commands to remove the `-d` flag (to be consistent with the instructions above), and add another step to open a second terminal window.

Reply:
I went ahead and added the `docker compose build` command, as I don't think it hurts (though I don't think it's needed).

Reply:
I stand corrected, it is indeed not needed.